### PR TITLE
LabWork1_AleksandraMihaylova

### PR DESCRIPTION
**Pull request by aleksandramihaylova**

### CODE REVIEW

File: LabWork1_AleksandraMihaylova/Script_LabWork1_AleksandraMihaylova.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+alter table employees
+add salary decimal(7,2),
+add previous_experience ENUM("yes", "NO");	
+
+alter table departments
+add manager_id int,
+add location varchar(255),
+add foreign key (manager_id) references managers (manager_id);
+
```
Comment (sslavova) [inline LabWork1_AleksandraMihaylova/Script_LabWork1_AleksandraMihaylova.sql:26]: Just add not null constraint in email and remove default value of hire_date

**Review by sslavova — CHANGES_REQUESTED** [review]
Just a small update on managers table

File: LabWork1_AleksandraMihaylova/Script_LabWork1_AleksandraMihaylova.sql (hunk 1 above)
Comment (aleksandramihaylova) [reply LabWork1_AleksandraMihaylova/Script_LabWork1_AleksandraMihaylova.sql:28]: I edited it. Thank you!

File: LabWork1_AleksandraMihaylova/Script_LabWork1_AleksandraMihaylova.sql [hunk 2]
```diff
@@ -10,14 +10,16 @@ create table if not exists employees
 first_name varchar(255) not null,
 last_name varchar(255) not null,
 department_id int,
-email varchar(255) unique,
-hire_date date default '2023-01-01',
+email varchar(255) not null unique,
+hire_date date not null,
 foreign key (department_id) references departments (department_id)
 );
```
Comment (sslavova) [inline LabWork1_AleksandraMihaylova/Script_LabWork1_AleksandraMihaylova.sql:16]: The hire date in employees table must have a default value.

**Review by sslavova — CHANGES_REQUESTED** [review]

**Review by sslavova — APPROVED** [review]

**Pull request merged**